error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StringRules.java

```diff
@@ -222,6 +222,19 @@ Function<Object, String> after() {
     }
   }
 
+  /** Prefer empty string literals over creating new empty String objects. */
+  static final class EmptyStringLiteral {
```

Comment:
We don't really have a good heuristic for the ordering of rules inside a rules collection, but roughly speaking right now the simplest / "most fundamental" go at the top. Let me move it there.

(Perhaps at some point we'll just automatically order them lexicographically 😅.)